tests/integration.rs

```diff
@@ -85,6 +85,38 @@ fn test_sec1_vs_pkcs8() {
     }
 }
 
+#[test]
+fn test_key_getter() {
```

Comment:
I think for this small number of test cases you have the right instinct leaving some repetition in place rather than doing something more complicated with `macro_rules` :+1: 

I wish it was a little bit easier to use `include_bytes!` in a "table-driven" way :-(

Reply:
Bikeshedding a bit, but I would do it like this:

```rust
#[test]
fn test_key_getter() {
    const P256: &[u8] = include_bytes!("data/nistp256key.pem");
    const RSA1025: &[u8] = include_bytes!("data/rsa1024.pkcs1.pem");
    ..

    fn key(bytes: &[u8]) -> Option<Vec<u8>> {
        let mut reader = BufReader::new(&data[..]);
        let item = rustls_pemfile::read_one(&mut reader).unwrap().unwrap();
        item.key()
    }

    assert!(key(P256).is_some());
}

```